qa-pipelines/tasks/cf-deploy.sh

```diff
@@ -127,6 +123,7 @@ if pxc_pre_upgrade; then
     helm upgrade --reuse-values scf ${CAP_DIRECTORY}/helm/cf/ \
         --namespace "${CF_NAMESPACE}" \
         --timeout 600 \
+        --set "secrets.UAA_CA_CERT=$(get_uaa_ca_cert)"
```

Comment:
backslash

Reply:
Prefer removing this; reuse-values makes it ok to not include it, and when we go back to testing embedded UAA we don't want the get_uaa_ca_cert function called because it will break